app/controllers/api/v1/restrooms.rb

```diff
@@ -11,6 +10,10 @@ class Restrooms < Grape::API
       resource :restrooms do
         desc "Get all restroom records ordered by date descending."
         params do
+          use :pagy,
+              items_param: :per_page,
+              items: 10,
+              max_items: 100
```

Comment:
I noticed these lines are being reused in each desc, would it be possible to set this param for all of them outside of each example? Kinda like line 5 on the old file.

Something like `use :page, items_param :per_page, items: 10, max_items: 100` on line 5 ofthis file.

Reply:
Yes, I have changed it so that those parameters are written as a class constant for the descs to reference instead and have [committed that change](https://github.com/RefugeRestrooms/refugerestrooms/pull/681/commits/bf7126f4754375b161daf8f4ad4964fc56718701). Grape Pagy does not appear to have anything equivalent to the old line 5, so this would be the best way to extract those parameters.